src/core/status.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+// Possible IPFS status values:
+module.exports = {
+  new: 'new',
+  initializing: 'initializing',
+  initialized: 'initialized',
+  starting: 'starting',
+  running: 'running',
+  stopping: 'stopping',
+  stopped: 'stopped'
+}
```

Comment:
Right now, in order to follow the sequence of booting a node, we need to open 4+ files. 

Let's put this into the state.js.